core/src/test/resources/replica-test.properties

```diff
@@ -0,0 +1,41 @@
+
+#
+# Copyright 2016 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+address=localhost:5000
+
+replica.1=localhost:5000
+replica.2=localhost:5001
+replica.3=localhost:5002
+
+transport=io.atomix.catalyst.transport.NettyTransport
+transport.threads=1
+
+quorumHint=3
```

Comment:
Since these are specific settings for the raft / cluster part perhaps prefixing them with a name would make sense?
like `cluster.quoremHint`